Formula/otp-cli.rb

```diff
@@ -0,0 +1,26 @@
+class OtpCli < Formula
+  desc "Tool for generate otp code in terminal"
+  homepage "https://github.com/chyroc/otp-cli"
+
+  url "https://github.com/chyroc/otp-cli/releases/download/v0.2.0/otp-cli-0.2.0.tar.gz"
+  sha256 "7e223d86e924e95e722b6c2b69acd0a4963085df2726132fcc6c0731caa93150"
+  head "https://github.com/chyroc/otp-cli"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
```

Comment:
This isn't allowed in go 1.16 anymore. The software should just build using `go build *std_go_args`

Reply:
fixed.